### ISSUE TITLE
Patents and Licenses

### ISSUE THREAD

**Issue by TheJaredWilcurt** [AOMediaCodec/av1-avif#2]
One of the primary concerns for adopting new image formats is if they are patent encumbered or have non-compatible licenses with what businesses need. The current documentation does not mention patents or licenses. 

* https://aomediacodec.github.io/av1-avif/

Even if this is a decision that is planned to be made after the format becomes more stable, that should be mentioned.

**Comment by PauKerr** [comment]
What I will do at this point is to add the same link that is found in the AV1 Bitstream and Decoder specification to reference the AOM license: https://aomedia.org/license/. Otherwise, there is no more explicit reference in that specification to the intent to produce an unencumbered codec.

**Comment by TheJaredWilcurt** [comment]
Just make sure it can be found with `CTRL+F` and either `patent` or `license` on the page.

**Comment by novomesk** [comment]
AVIF specification references HEIF standard ( ISO/IEC 23008-12:2017 standard).
Nokia sent a patent declaration with reference to the standard ( https://isotc.iso.org/livelink/livelink/fetch/2000/2122/3770791/16231513/2015-12-08_NokiaTechnologiesOy__23008-12.pdf ), mentioning among others some 14/583332 patent called _Media encapsulating and decapsulating_.
Are we sure that the Nokia's patent doesn't cover structure of the .AVIF files or the license is granted?

**Comment by mirh** [comment]
Indeed, I cannot believe it, but yet the document with that wording is there. 
And there's like another half a dozen patent claims related to ISO/IEC 23008-12 in the database [here](https://www.iso.org/iso-standards-and-patents.html).
